addons/stock/tests/test_stock_flow.py

```diff
@@ -27,12 +27,12 @@ def setUpClass(cls):
         cls.uom_dozen = cls.UomObj.create({
```

Comment:
We already define `cls.uom_dozen` in  `TestStockCommon` (by getting the existing one instead of creating a new one like here but it should be better to stick to one way and not reset the property in the setup)

Reply:
I agree, I removed this one :+1: